x-pack/plugins/ml/public/application/routing/routes/datavisualizer/index_based.tsx

```diff
@@ -43,19 +36,15 @@ export const indexBasedRouteFactory = (
   ],
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { redirectToMlAccessDeniedPage } = deps;
-
-  const { index, savedSearchId }: Record<string, any> = parse(location.search, { sort: false });
-  const { context } = useResolver(index, savedSearchId, deps.config, deps.dataViewsContract, {
-    checkBasicLicense,
-    cacheDataViewsContract: () => cacheDataViewsContract(deps.dataViewsContract),
-    checkGetJobsCapabilities: () => checkGetJobsCapabilitiesResolver(redirectToMlAccessDeniedPage),
-  });
+const PageWrapper: FC = () => {
+  // TODO check if required capabilities is correct
+  const { context } = useRouteResolver('basic', ['canGetJobs']);
```

Comment:
'basic' is correct 👍

Reply:
@qn895 I'm not sure about `canGetJobs` here, cc @jgowdyelastic